app/src/main/res/values/strings.xml

```diff
@@ -90,6 +90,9 @@
     <string name="msg_not_booking">You still not booking yet</string>
     <string name="title_na">N/A</string>
     <string name="msg_book_to_edit">You can book again to edit your last booking</string>
+    <string name="title_canceled">Canceled</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.title_canceled` appears to be unused
 Unused resources make applications larger and slow down builds.